### PR TITLE
Create automated-testing-with-a-custom-driver.md

### PR DESCRIPTION
**Pull request by pfrazee**
@zeke asked me to write up a quick tutorial on how you can write an alternative to spectron for driving tests. Here it is, LMK if you'd like any changes.

### CODE REVIEW

File: docs/tutorial/automated-testing-with-a-custom-driver.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  
+  // define your RPC-able methods here
+}
+```
+
+Then, in your test suite, you can use your test-driver as follows:
+
+```js
+var test = require('ava')
+var electron = require('./node_modules/electron')
```
Comment (zeke) [inline docs/tutorial/automated-testing-with-a-custom-driver.md]: Why `./node_modules/electron` and not `electron`? So you can get the path? If so, a var name like `electronPath` might be more clear.

**Comment by zeke** [comment]
Also it looks like the JS is invalid in a few spots:

```
   tutorial/automated-testing-with-a-custom-driver.md
         12:17:    Parsing error: Unexpected token }
         93:59:    Infix operators must be spaced.
         111:4:    Unexpected trailing comma.
         112:1:    Trailing spaces not allowed.
``

**Comment by pfrazee** [comment]
@zeke done

**Comment by pfrazee** [comment]
Done!

**Comment by pfrazee** [comment]
@codebytere Whoop! Any ETA for it going on the site?

**Comment by codebytere** [comment]
@zeke would have a better idea than me 😁

**Pull request merged**